test/read.test.ts

```diff
@@ -143,11 +135,10 @@ describe("commands/read", function () {
       .parse();
     assert.calledWith(
       consoleLog,
-      `[
```

Comment:
I see what you're doing here and elsewhere, but I think we actually want to test that the string output is as expected. We actually _want_ to test that it is the json string we are expecting. Rather than re-parsing as you have here... though you can try to convince me otherwise?

Reply:
I'm logging objects instead of strings now. Strings don't get highlighting with logged out to the shell, but objects do.

Reply:
Ah ok. For deeply nested objects, this will be an issue (which is why I did it that way). Is it possible to console.log and specify a depth when stringifying like in JSON.stringify?

Reply:
You can use `console.dir(output, {depth: null});`